### ISSUE TITLE
push publiccode.yml to new branch on form submit, and PR it

### ISSUE THREAD

**Issue by ckingbailey** [ckingbailey/publiccode-pusher#7]
Push publiccode.yml to new branch created by flow described on issue #6

### Commit the file to the branch
PUT `https://api.github.com/repos/:owner/:repo/contents/publiccode.yml`, body:
```JSON
{
  "message": "Add publiccode.yml",
  "content": "{ base-64 encoded yml file contents... }",
  "branch": "Publiccode-Pusher/add-publiccode-yml"
}
```
more here https://developer.github.com/v3/repos/contents/#create-or-update-a-file

### Open a new Pull Request
POST `https://api.github.com/repos/:owner/:repo/pulls`, body:
```JSON
{
  "title": "Add publiccode.yml describing this project",
  "head": "Publiccode-Pusher/add-publiccode-yml",
  "base": "master",
  "body": "{ I'll come up with this nice PR copy later. Specify that this PR was opened by a machine. Link to the brigade project index project and maybe Slack channel, too. }"
}
```

**Comment by ckingbailey** [comment]
Possible errors:
422 Unprocessable Entity:
```
"message": "Reference already exists",
"documentation_url": "https://developer.github.com/v3/git/refs/#create-a-reference"
```

**Comment by ckingbailey** [comment]
resolved by #21
